plugins/integrations/kubernetes-service/src/main/java/com/cloud/kubernetes/cluster/KubernetesClusterVmMap.java

```diff
@@ -0,0 +1,27 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+package com.cloud.kubernetes.cluster;
+
+/**
+ * KubernetesClusterVmMap will store a map of ID of KubernetesCuster
+ * and ID of its VirtualMachine
```

Comment:
English could improve: I am reading "KubernetesClusterVmMap will store a one-to-one map of the ID of a KubernetesCuster and the ID of its VirtualMachine", correct? or can it m-to-n or 1-to-n or m-to-1?

Reply:
@DaanHoogland KubernetesClusterVmMap points to a single entry of cluster ID to VM ID map.
A single cluster can have multiple VM and therefore a cluster will have multiple KubernetesClusterVmMapVOs
A particular VM is tied to only a single cluster

Reply:
ok, please rephrase your javadoc to reflect that, than.

Reply:
Done